src/controllers/notifications.js

```diff
@@ -1,67 +0,0 @@
-'use strict'
```

Comment:
From @emschwartz : Is there a way to add the apidoc comments back but for the websocket connection?

The apidoc comments for the websocket connection are available at https://github.com/interledger/five-bells-ledger/blob/ws_notifications/src/controllers/accounts.js#L126